### ISSUE TITLE
Sound Effects

### ISSUE THREAD

**Issue by grady404** [cooper-anderson/Blockade#53]
Sound effects would include:
- sliding sound as the block is moving
- soft "thud" sound when block hits a wall/other object
- goal/win sound
- sounds for when you switch/select menu items

**Comment by grady404** [comment]
Uh, okay, I'm actually way worse at creating sound effects than I thought. I can't make the sliding sound, or the thud sound. But I'd be able to create sounds that are closer to a tone or a chime than a drum hit, for example the level win sound. So for the ones I can't create, we could either try to find someone who is able to make them, or just use royalty-free stock sounds (which I'd really hate to do).

**Comment by cooperuser** [comment]
I'll look into it as well.

**Comment by grady404** [comment]
Alright, so I found some sound effects online that might give us inspiration on how to make our own sliding sound (no, we're not going to steal these ones). I think something like this would be the ideal sliding sound, and would be easy to reproduce:

[`Sound 1`](https://www.youtube.com/watch?v=GUej1qNZ2Wg) [`Sound 2`](https://www.youtube.com/watch?v=ZbGjlWfyPxU)

If not, here's an alternative you might consider that would also be very easy to reproduce (although we would need to sustain the sound for longer):

[`Sound 3`](https://www.youtube.com/watch?v=L9hluUUdKf4)

And here are a couple of "questionable" alternatives that somehow might be worth looking into, just for shits and giggles:

[`Sound 4`](https://www.youtube.com/watch?v=XoeTG4ioRSU) [`Sound 5`](https://www.youtube.com/watch?v=-RtMzdrnR48)

Whatever we choose, I think it would be in our interest to record multiple variations of the same sound, so it doesn't get repetitive and obnoxious. I would suggest around 5 to 8 variations, but we could always do more if we were feeling up to it.

**Comment by cooperuser** [comment]
Interesting input. A softer version of [`Sound 1`](https://www.youtube.com/watch?v=GUej1qNZ2Wg) is what I had thought of when I first considered sounds, but I think that we could definitely make something of this.

**Comment by grady404** [comment]
All we'd need to do to reproduce a sound like that would be to take some sort of stone or granite slab, and slide it along concrete. I'm sure there's something lying around the convent building at school

**Comment by grady404** [comment]
The only thing I can't seem to find is a "thud" sound, I guess we would need to improvise that one

**Comment by cooperuser** [comment]
Yeah, but I'm sure it won't be that hard to create a "thud" sound

**Comment by grady404** [comment]
But the type of thud we're looking for is very specific

**Comment by cooperuser** [comment]
True. We need a sort of soft thud with a quick quiet echo

**Comment by grady404** [comment]
Btw did you re-implement the box move buttons yet?

**Comment by cooperuser** [comment]
No, but I was working on it a bit after school

**Comment by grady404** [comment]
When you commit it just be sure not to undo the commit I just added (`v0.2.34`)

**Comment by cooperuser** [comment]
Okay
